documentation/modules/cruise-control/proc-configuring-deploying-cruise-control.adoc

```diff
@@ -2,22 +2,25 @@
 //
 // assembly-cruise-control-concepts.adoc
 
-[id='proc-deploying-cruise-control-{context}']
-= Deploying Cruise Control
+[id='proc-configuring-deploying-cruise-control-{context}']
+= Configuring and deploying Cruise Control with Kafka
 
 [role="_abstract"]
-To deploy Cruise Control to your Strimzi cluster, define the configuration using the `cruiseControl` property in the `Kafka` resource, and then create or update the resource.
-
+Through `Kafka` resource configuration, the Cluster Operator can deploy Cruise Control when deploying a Kafka cluster.
+You can use the `cruiseControl` properties of the `Kafka` resource to configure the deployment.
 Deploy one instance of Cruise Control per Kafka cluster.
 
+Use `goals` configuration in the Cruise Control `config` to specify optimization goals for generating optimization proposals.
+You can use `brokerCapacity` to change the default capacity limits for goals related to resource distribution.
```

Comment:
@kyguy is there no documentation about overrides yet?

Reply:
I think this PR needs a rebase to pick up the new changes

Reply:
Rebased and got the overrides section in :+1: